app/sign-out/index.js

```diff
@@ -0,0 +1,14 @@
+// NPM dependencies
+const router = require('express').Router()
+
+// Local dependencies
+const { get } = require('./controllers')
+
+// Define routes
+router.get('/', get)
+
+// Export
+module.exports = {
+  router,
+  mountpath: '/sign-out',
```

Comment:
I think we should add this as a route to the `auth` feature rather than a separate feature. It feels linked to that.

Reply:
That was my preference too, but the middleware intercepts it first. The only alternative was to move the middleware inline.

Reply:
I’ve done some refactoring to when and how that middleware actually loads in #115 which should solve that. Maybe I’ll get that in then you can make the changes on top of that. 

You’ll just then need to add the sign out path to the auth whitelist.

Reply:
Ok great.